docs/misc/guide.md

```diff
@@ -4,6 +4,10 @@ Guide is a feature that allows us to write content on various topics like basics
 
 See the [tech spec](../specs/guide.md) for details and rationale behind the implementation.
 
+## How to update
+
+Change `GITBOOK_REVISION` constant in `packages/suite-data/src/guide/constants.ts`. Example: [40855097](https://github.com/trezor/trezor-suite/commit/408550979cd58e78df297e30fb32e45935529a80). That's it.
+
```

Comment:
I would move it under `## Content` section because without reading that section, it is not clear what this section says.

Reply:
^^ it's especially not clear what should I change it to.

Moving it *under* the `Content` section would help that. .)

Reply:
cc9cddc 

---

I really should stop micro-managing and just let you guys working, next time :innocent:.